stopcovid/dialog/models/state.py

```diff
@@ -45,15 +45,15 @@ def set_language(cls, value: Optional[str]) -> Optional[str]:
 class PromptState(pydantic.BaseModel):
     slug: str
     start_time: datetime.datetime
-    failures: Optional[int] = 0
-    reminder_triggered: Optional[bool] = False
+    failures: int = 0
+    reminder_triggered: bool = False
```

Comment:
I looped through the entire `dialog-state-prod` table and didn't find any cases where these values were `None`